test/shamap-test.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable max-len, valid-jsdoc */
+'use strict';
+
+var assert = require('assert');
+var SHAMap = require('../src/common/hashes/shamap').SHAMap;
+var TYPE_TRANSACTION_NM = require('../src/common/hashes/shamap').TYPE_TRANSACTION_NM
+
+var HEX_ZERO = '00000000000000000000000000000000' +
+               '00000000000000000000000000000000';
+
+function int_to_vuc(v) {
```

Comment:
convert this function, and the rest, to camel case to be consistent with the codebase.